### ISSUE TITLE
How to Control the Stability of Training Neural Networks With the Batch Size

### ISSUE THREAD

**Issue by jatinkumar762** [jatinkumar762/AI_Articles#18]
## TL;DR
How to Control the Stability of Training Neural Networks With the Batch Size
### Article Link
[Link](https://machinelearningmastery.com/how-to-control-the-speed-and-stability-of-training-neural-networks-with-gradient-descent-batch-size/)
